# Request
# An ordered list of object names
string[] objects

# An ordered list of classes of objects
uint8 CLASS_PROP=0
uint8 CLASS_CONTAINER=1
uint8 CLASS_SUPPORTER=2
uint8[] classes

# An ordered list of number of states
uint64[] num_states

# All learned factors
srp_md/Factor[] factors
---
# Response
string[] object1
string[] object2
string[] relation
